include/aikido/rviz/TSRMarker.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef AIKIDO_RVIZ_TSRMARKER_H_
+#define AIKIDO_RVIZ_TSRMARKER_H_
+
+#include <set>
+#include "FrameMarker.hpp"
+#include "SmartPointers.hpp"
+
+namespace aikido {
+namespace rviz {
+
+class TSRMarker {
+public:
+  TSRMarker();
+  ~TSRMarker(){};
+
+  void addFrameMarker(FrameMarkerPtr const &marker);
```

Comment:
I don't think this should be part of the public API. The simplest option would be to modify `TSRMarker`'s constructor to directly accept the `std::set<FrameMarkerPtr>`.